alfresco-authentication/backend/app/src/main/kotlin/com/ritense/valtimo/alfrescoauth/ValtimoApplication.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
```

Comment:
Please update all copyright headers and make sure they've been added to every java or kotlin file.